test/integration/test_dwarf.c

```diff
@@ -291,39 +291,39 @@ bool test_dwarf3_cpp_basic(void) { // this should work for dwarf2 aswell
 	check_abbrev_tag(DW_TAG_subprogram);
 	{
 		check_abbrev_children(true);
-		check_abbrev_count(8);
+		check_abbrev_count(7);
 	}
 	i++;
 	check_abbrev_tag(DW_TAG_formal_parameter);
 	{
 		check_abbrev_children(false);
-		check_abbrev_count(3);
+		check_abbrev_count(2);
 	}
 	i++;
 	check_abbrev_tag(DW_TAG_formal_parameter);
 	{
 		check_abbrev_children(false);
-		check_abbrev_count(2);
+		check_abbrev_count(1);
 	}
 	i++;
 	check_abbrev_tag(DW_TAG_member);
 	{
 		check_abbrev_children(false);
-		check_abbrev_count(5);
```

Comment:
Why these off-by-one differences?

Reply:
Because the previous count has an offset of +1